docs/en/serverless/apm/new-experience-services.mdx

```diff
@@ -0,0 +1,53 @@
+---
+slug: /serverless/observability/new-experience-services
+title: New Services inventory experience
+description: Try out our new Services inventory, which enables you to monitor service logs without instrumenting all your services.
+tags: [ 'serverless', 'observability', 'overview' ]
+---
+
+<DocBadge template="technical preview" />
+
+With this technical preview,
+Elastic expands the **Services** inventory to include services found in logs.
+This means you can monitor the services you care about even if you only have logs
+and aren't ready to instrument your services.
+
+The new **Services** inventory shows services instrumented with APM or OpenTelemetry,
+plus any services declared using `service.name` in your logs:
+
+{/* TODO: Replace the following image with a screen capture that shows realistic data */}
+<DocImage url="../images/entity-centric-services-inventory.png" alt="Services inventory that shows APM data and log data"/>
+
+For instrumented services that also have custom logs, the APM and log data are displayed in the same view.
+For services that only have log data,
+the log data is displayed, and you have the option of adding APM later.
+
+<DocCallOut title="Note">
+   The new **Services** inventory requires the Elastic Entity Model (EEM). To learn more, refer to <DocLink slug="/serverless/observability/elastic-entity-model" />.
+</DocCallOut>
+
+## Turn on the new experience
+
+The new Services inventory is turned off by default. To turn it on:
+
+1. Go to **Applications** → **Services**.
+1. Click **Try out Elastic's new experience!**
+
+If you have **Admin** privileges, the Elastic Entity Model will be turned on automatically.
+Otherwise, you need to ask someone with **Admin** privileges to turn on this feature under **Stack Management**.
+
+## Add services to the inventory
+
+To add services to the inventory, use one of the following approaches:
+
+* Instrument your services with APM or OpenTelemetry.
+To learn how, refer to <DocLink slug="/serverless/observability/apm-send-data-to-elastic" />.
+* Declare `service.name` in your logs.
+To learn how, refer to LINK TBD. {/* TODO: Add link to topic that Mike is working on.*/}
```

Comment:
Looks like https://github.com/elastic/observability-docs/pull/4054 defines the slug as /serverless/observability/add-logs-service-name.